app/lib/actions/pulp/repository/sync.rb

```diff
@@ -21,7 +21,7 @@ def invoke_external_task
             overrides[:validate] = !(SETTINGS[:katello][:pulp][:skip_checksum_validation])
             overrides[:options] = input[:options] if input[:options]
             repo = ::Katello::Repository.find_by(:pulp_id => input[:pulp_id])
-            output[:pulp_tasks] = repo.backend_service(smart_proxy).sync(overrides)
+            output[:pulp_tasks] = repo.backend_service(::SmartProxy.pulp_master).sync(overrides)
```

Comment:
This sync is also against the main server and never a proxy?

Reply:
smart proxies are synced via a different class:  https://github.com/Katello/katello/blob/master/app/lib/actions/pulp/consumer/sync_capsule.rb#L16

we could look at combining them eventually, but today they are not